.changelog/unreleased/features/1409-vega-protos.md

```diff
@@ -1,3 +1,2 @@
 - Support for compatibility with gaia Vega upgrade (protos matching ibc-go
-  v1.2.2 and SDK v0.44.3) ([#1409](https://github.com/informalsystems/ibc-
-  rs/issues/1409))
\ No newline at end of file
+  v1.2.2 and SDK v0.44.3) ([#1409](https://github.com/informalsystems/ibc-rs/issues/1409))
```

Comment:
IMHO, the wrong PR is linked here. It should probably be #1408.

Reply:
Good catch! 🙏